exchanges/gdax.js

```diff
@@ -307,6 +307,22 @@ Trader.prototype.getMaxDecimalsNumber = function (number, decimalLimit = 8) {
   return decimalCount <= decimalLimit ? decimalNumber.toString() : (Math.floor(decimalNumber * decimalMultiplier) / decimalMultiplier).toFixed(decimalLimit);
 };
 
+Trader.prototype.deductFee = function (number) {
```

Comment:
@askmike not sure if it's necessary, but I think I had to deduct the Fee here to prevent the `price too precise` error. Do you know when the fee (should) be deducted? is it on the GDAX side? Or somewhere in Gekko?